tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -203,3 +203,23 @@ def test_first_non_empty_item_no_valid(self):
         list_to_filter = [None, None]
 
         assert self.transforms.first_non_empty_item(list_to_filter) == ""
+
```

Comment:
There should be unit tests for the english and welsh language versions of england/wales too.

Reply:
Wales/England schemas are not transformed here👍

Reply:
As mentioned, we should name the method more appropriately as this is currently unclear.

Reply:
Done, thanks!